README.adoc

```diff
@@ -262,6 +262,13 @@ auth:
   strategy: VALUE
 ----
 
+|`KIALI_INSTALLATION_TAG`
+|Tag used to identify a particular instance/installation of the kiali server (Default: kiali)
```

Comment:
Based on line 205 of config.go, the default is "Kiali" (capitalized) not "kiali".
However, I see in the kiali-ui PR that it seems to be using "Kiali Console" - see https://github.com/kiali/kiali-ui/pull/1079/files#diff-b7bd986e1ec3d4168360ce3ad6d59e34R46

I think we should maintain some consistency here rather than diverging default values.

Reply:
Ok. So I removed the default from the server. The default will be set as "Kiali Console" in the UI if the UI does not receive any value from the server.